src/components/thumbnailcolumn/thumbnailcolumn.scss

```diff
@@ -5,25 +5,26 @@
 
     $thumbnail-width: 220px;
     $thumbnail-inner-width: 204px;
-    
+
     $project-height: 208px;
     $gallery-height: 164px;
-    
+
     margin: 0 auto;
     padding: 12px 0;
     justify-content: flex-start;
     flex-direction: column;
 
     .thumbnail {
         margin: 7px;
-        border-radius: 4px;
-        box-shadow: 0 0 0 1px $active-gray;
-        background-color: $ui-white;
-        padding-bottom: 4px;
+        // unnecessary:
+        // padding-bottom: 4px;
```

Comment:
`//` is invalid CSS, you should use `/* */` instead. I think SCSS might allow it but we shouldn't depend on that.  Also if it's unnecessary, you can just delete it :)

Reply:
I forgot to remove the comments I put there when experimenting ._. will remove them